### ISSUE TITLE
Calling FrameConvexPolygon2D.setIncludingFrame() doesn't set frame with empty vertex.

### ISSUE THREAD

**Issue by rjgriffin42** [ihmcrobotics/euclid#16]
The following case current fails:

There are two random polygons, in random frames, `polygonA` and `polygonB`.

```
polygonA.clear();
polygonB.clear();
polygonB.setIncludingFrame(polygonA);
assert(polygonA.epsilonEquals(polygonB, 1e-5);
```

This fails because of a reference frame mismatch.

**Comment by rjgriffin42** [comment]
Note that a way around this is

```
polygonA.clear();
polygonB.clear();
polygonB.setIncludingFrame(polygonA.getReferenceFrame(), polygonA);
assert(polygonA.epsilonEquals(polygonB, 1e-5);
```

This isn't exactly a desirable work around, though.

**Comment by SylvainBertrand** [comment]
Oh wow, I'll fix that, thanks!

**Comment by SylvainBertrand** [comment]
Addressed in a2ac89557e333494b2358564d604657c19d1cd9b
